pii_recognition/recognisers/comprehend_recogniser.py

```diff
@@ -13,10 +13,31 @@
 AWS_REGION = "us-west-2"
 
 
+class ModelMapping(dict):
+    def __getitem__(self, key: str) -> Callable:
+        try:
+            return super().__getitem__(key)
+        except KeyError:
+            key_list = list(super().keys())
+            raise ValueError(
+                f"Available model names are: {key_list} but got model named {key}"
+            )
+
+
 class ComprehendRecogniser(EntityRecogniser):
-    def __init__(self, supported_entities: List[str], supported_languages: List[str]):
+    def __init__(
+        self,
+        supported_entities: List[str],
+        supported_languages: List[str],
+        model_name: str,
+    ):
         sess = config_cognito_session(IDENTITY_POOL_ID, AWS_REGION)
-        self.comprehend = self._initiate_comprehend(sess)
+        comprehend = self._initiate_comprehend(sess)
+        model_mapping = ModelMapping(
+            ner=comprehend.detect_entities, pii=comprehend.detect_pii_entities
+        )
+        self.model_func = model_mapping[model_name]
```

Comment:
This is the most recent change being made. Before this change made, if users want to make predictions with a model neither ner nor pii, it won't fail until actual calling predict. Now, if a wrong model being passed, it will fail at `__init__`.